### ISSUE TITLE
x/website: please return current golang version to website first page

### ISSUE THREAD

**Issue by bfreis** [golang/go#33083]
The previous website design had a string with the current version. As part of my workflow, I used to check the website to see if a new version was released, at the same time I was checking if new interesting blog posts were available.

Seems like the new website removed that version string.

Could you please put it back?

Thanks!

**Comment by andybons** [comment]
If the version were near the download button (as opposed to being in the footer) would that satisfy your request?

**Comment by bfreis** [comment]
Definitely! Anywhere would be fine, and near the Download button makes a lot of sense.
Thanks!

**Comment by gondar00** [comment]
@andybons I am interested in contributing can I pick this up?

**Comment by andybons** [comment]
@gondar00 let me get a design first then I’ll post it here.

**Comment by dmitshur** [comment]
This is implemented on [go.dev](https://go.dev). Perhaps that can help inform this issue.
